### DISCUSSION TITLE
Where can I find the Sunbird Knowledge Platform local setup documentation?

### DISCUSSION THREAD

**Discussion by joffinjoy** [Sunbird-inQuiry/Community#17] [Q&A] [answered]
@krgauraw @Krishnaj20 
I remember there was a KP local setup documentation that was being written by Gauraw. I am no longer able to find it. Please provide me with the links for the same.

**Comment by Krishnaj20** [comment, marked as answer]
@joffinjoy - Please go through the developer installation steps in the respective building block microsite. 
@aks30 - Please assist Joffin.
